wiki/Buddy-configuration.md

```diff
@@ -41,6 +41,22 @@ Note: in the Buddy's configuration the screen is referred to as a display due to
 
 Resolution change not supported yet as the [API](https://wayland.app/protocols/wlr-output-management-unstable-v1) is not mature enough and the compositor support is lacking.
 
+*Note:* Although we cannot change the resolution through sunshine and, we can nicely ask the ask the DE compositor to set through the sunshine *Do* and *Undo* actions.
```

Comment:
Should we maybe refer to https://docs.lizardbyte.dev/projects/sunshine/en/latest/about/guides/app_examples.html#changing-resolution-and-refresh-rate instead?

Reply:
Oh yeah! I was unaware of this documentation. 
I'll update with a link instead